src/pretix/control/views/subevents.py

```diff
@@ -103,6 +103,9 @@ def delete(self, request, *args, **kwargs):
             messages.error(request, pgettext_lazy('subevent', 'A date can not be deleted if orders already have been '
                                                   'placed.'))
             return HttpResponseRedirect(self.get_success_url())
+        if self.get_object().event.subevents.count() == 1:  # checking if this is the last date in the event series
+            messages.error(request, pgettext_lazy('subevent', 'Last date of an event series can not be deleted'))
```

Comment:
If we put "The last date […] can not be deleted." (with a dot at the end and a The in the beginning) it looks a bit better, I guess :)

Reply:
sounds good.